py-polars/polars/series/series.py

```diff
@@ -3914,6 +3927,32 @@ def is_utf8(self) -> bool:
         """
         return self.dtype is Utf8
 
+    def is_object(self) -> bool:
```

Comment:
I don't want these methods on the Series object. I want to phase out the others too (e.g. `is_utf8`, `is_integer`, ...). You should check the `dtype` directly.

Reply:
I can make follow up PR's in which I phase them out by checking dtype directly, just like I did in my last change. I would then make a separate PR per dtype, is that something you would like to see?

Reply:
@stinodego ?

Reply:
These should be methods on the datatype, e.g. `Series.dtype.is_struct()`. But I do not want these added right now as I need to change the implementation of the DataType classes first.

So for the purposes of this PR, the way you have it right now is fine!